ns1/ns1.go

```diff
@@ -0,0 +1,30 @@
+package ns1
+
+import (
+	"errors"
+
+	"github.com/mholt/caddy/caddytls"
+	"github.com/xenolf/lego/acme"
+	"github.com/xenolf/lego/providers/dns/ns1"
+)
+
+func init() {
+	caddytls.RegisterDNSProvider("ns1", NewDNSProvider)
+}
+
+// NewDNSProvider returns a new MyProvider DNS challenge provider.
```

Comment:
What is "MyProvider"?

Reply:
I copy pasted from the sample in the wiki (https://github.com/mholt/caddy/wiki/Writing-a-Plugin:-DNS-Provider) and forgot to change the comment after adapting the code.